### PR TITLE
Adding config setting for loading fonts from google (default true) or locally (false)

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

**Comment by jeffswartz** [comment]
This PR adds a config setting to determine whether to load the “Open Sans” font from Google (the default) or from the server hosting OpenTok demo: https://github.com/opentok/opentok-rtc/pull/493.

To use this feature, you can either:

* Set the `USE_GOOGLE_FONTS` env variable to false (and the client will load fonts from the server hosting OpenTok demo).
* Set `useGoogleFonts` to `true` in the config.json file (and the client will load fonts from the server hosting OpenTok demo).
* Leave the settings unset (and the client will load fonts from Google, the default).

**Comment by normanargueta** [comment]
One more thing, are we going to keep the Open Sans fonts in the repository? I thought the idea was to remove all the font files from the repository and use a CDN to serve them

File: web/less/fonts.less [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 /* open-sans-regular - latin */
 @font-face {
-  font-family: 'Open Sans';
+  font-family: if(loadFontsFromAppServer(), 'Open Sans', '');
```
[the remark this answers is not in the document]
Comment (jeffswartz) [reply web/less/fonts.less]: Yes, that would be better, @normanargueta. I'll add a commit to do this.

**Comment by jeffswartz** [comment]
@normanargueta, we have decided to keep Open Sans in the repository (and use the config flag to determine whether to load it from google or from the local server).

**Pull request merged**